### PR TITLE
fix initiliazed => initialized

### PR DESCRIPTION
**Pull request by BingGuanqi**
fix initiliazed => initialized

Signed-off-by: Guanqi Bing <guanqi.bing@daocloud.io>

### CODE REVIEW

**Comment by yeya24** [comment]
Thanks for your contribution. Could you sign off your commit please? You can check [here](https://github.com/dragonflyoss/Dragonfly/pull/762/checks?check_run_id=182593374).

File: dfget/core/uploader/peer_server_executor.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func StartPeerServerProcess(cfg *config.Config) (port int, err error) {
 	if defaultExecutor != nil {
 		return defaultExecutor.StartPeerServerProcess(cfg)
 	}
-	return 0, fmt.Errorf("executor of peer server isn't be initiliazed")
+	return 0, fmt.Errorf("executor of peer server isn't be initialized")
```
Comment (yunfeiyangbuaa) [inline dfget/core/uploader/peer_server_executor.go]: `executor of peer server hasn't been initialized `may be better,WDYT？

Comment (allencloud) [reply dfget/core/uploader/peer_server_executor.go]: I do think so. Could you help to fix that? @BingGuanqi

**Comment by Starnop** [comment]
LGTM.

File: dfget/core/uploader/peer_server_executor.go (hunk 1 above)
Comment (BingGuanqi) [reply dfget/core/uploader/peer_server_executor.go]: sure. I have already fixed that

**Pull request merged**